.github/workflows/tests.yaml

```diff
@@ -0,0 +1,17 @@
+name: Tests
```

Comment:
Any thoughts on if we should have a seperate test workflow for Go and Typescript ? Or the same workflow ? If so we should name this workflow (Go) or name the job on this workflow `test-go` :)

Reply:
Perhaps for now, let's just keep it simple and be consistent with linting. So name this workflow "Test" and then have a job named "Test Go":
![image](https://github.com/strideynet/bsky-furry-feed/assets/16336790/e70f56b2-d595-4747-bc7c-1fbf3693d424)

Reply:
I don't know Actions well enough to know the consequences of that. Since we already have a **Lint / Lint Go** workflow and I like that, I’m happy with using the same pattern here.